<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateMasterListWindow</class>
 <widget class="QDialog" name="UpdateMasterListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdateMasterListWindow</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 사진 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Storage 이름 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Container ID : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 이름 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 유형 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 용도 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>구매일 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>물품 갯수 : </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>181</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>590</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>물품 Update 정보 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>274</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Item 정보 검색하러 가기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>현재 Storage, Container 보기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>함초롬바탕</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Storage, Container 정보 추가</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton_2</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>viewStoCon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton_2</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>addStoCon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homeButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>updateItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateMasterListWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>604</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>view()</slot>
  <slot>viewStoCon()</slot>
  <slot>addStoCon()</slot>
  <slot>exit()</slot>
  <slot>updateItem()</slot>
  <slot>clear()</slot>
 </slots>
</ui>
